### DISCUSSION TITLE
[ICEBRG] How to set write.object-storage.enabled property for table?

### DISCUSSION THREAD

**Discussion by sdasdasd1** [trinodb/trino#23222] [Q&A]
I want to create iceberg table with that option
write.object-storage.enabled"
But I cant provided in WITH statement, getting an error -  table property 'write.object-storage.enabled' does not exist

**Comment by ebyhr** [comment]
https://github.com/trinodb/trino/pull/20555 is going to support `write.object-storage.enabled` property. 

Note that the connector doesn't support specifying arbitrary table properties: https://github.com/trinodb/trino/pull/20393
